package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "uw-frame",
-  "version": "2.8.0",
+  "version": "2.9.0",
```

Comment:
We don't have to update the package.json, it's gets updated as part of our release process with a version that matches the java release as well.  Eventually, it would be cool to doing auto releasing.